IGCexpansion/CodonGeneconv.py

```diff
@@ -384,15 +403,21 @@ def get_MG94Geneconv_and_MG94(self):
                         continue
                     sc = self.codon_to_state[cc]
                     # (ca, cb) to (ca, cc)
-                    Qb = Qbasic[sb, sc]
+                    if isHomogenizing(ca, cc, self.codon_table) and self.use_Homo_Omega():
+                        Qb = Qbasic_Homo[sb, sc]
+                    else:
+                        Qb = Qbasic[sb, sc]
                     if Qb != 0:
                         row.append((sa, sb))
                         col.append((sa, sc))
                         rate_geneconv.append(Qb)
                         rate_basic.append(0.0)
 
                     # (ca, cb) to (cc, cb)
-                    Qb = Qbasic[sa, sc]
+                    if isHomogenizing(cc, cb, self.codon_table) and self.use_Homo_Omega():
+                        Qb = Qbasic_Homo[sb, sc]

```

Comment:
This is careless, fix.

Reply:
Better read the difference via `git diff` before committing

Reply:
Sorry, I went too fast when I read the difference.